image/image.go

```diff
@@ -122,6 +148,31 @@ type genericManifest interface {
 	GetLayers() []string
 }
 
+type descriptor struct {
+	MediaType string `json:"mediaType"`
+	Size      int64  `json:"size"`
+	Digest    string `json:"digest"`
+}
+
+type manifestSchema2 struct {
+	Config descriptor   `json:"config"`
+	Layers []descriptor `json:"layers"`
+}
+
+func (m *manifestSchema2) String() string {
+	// TODO: does it make sense to have string here?
```

Comment:
there's a notable comment in there
```
LayersCommand(layers ...string) error // configure download directory?
```
why don't we extend layercommand to receive a directory name for destination?